### DISCUSSION TITLE
Problems occurred during version upgrade

### DISCUSSION THREAD

**Discussion by MrZmcH** [eclipse-theia/theia#11451] [Q&A]
Due to the use of @theia/ Toolbar, the package only appeared in V1.23.0. I upgraded from V1.20.0 to V1.23.0, but something unexpected happened, and there were no exceptions at V1.20.0.
Appears during page loading
![image](https://user-images.githubusercontent.com/45864434/180382755-448129f4-1852-44c5-9557-734f93932750.png)
It has been in the offline state and continues to fail to establish a connection
![image](https://user-images.githubusercontent.com/45864434/180383622-001dea7e-68cb-4203-8f2e-cc958744f193.png)

**Comment by MrZmcH** [comment]
I have overwritten the loading page and the original one still appears on the right.

**Comment by vince-fugnitto** [comment]
@MrZmcH 

> 1. https://github.com/eclipse-theia/theia/discussions/11451#discussion-4237187

I haven't seen the issue you reported but I believe you previously mentioned running in docker. It is possible you encountered https://github.com/eclipse-theia/theia/issues/11296 but some additional investigation and troubleshooting would be required.

> 2. https://github.com/eclipse-theia/theia/discussions/11451#discussioncomment-3203884

It is difficult to help without seeing any code but it sounds like you are not properly overriding the previous content. For instance, in the [example](https://github.com/eclipse-theia/theia-blueprint/blob/master/applications/electron/resources/preload.html) provided we explicitly remove the previous loading content:

- https://github.com/eclipse-theia/theia-blueprint/blob/ae6104e1e5a9d1956349d3dccd5b6079eab3db6a/applications/electron/resources/preload.html#L11-L18

**Reply by MrZmcH** [reply]
#11296
Same question,Still not solved, what could be the reason?
